Sources/VaporOAuth/Models/OAuthDiscoveryDocument.swift

```diff
@@ -0,0 +1,82 @@
+import Vapor
+
+public final class OAuthDiscoveryDocument: Content {
```

Comment:
same here (`struct`?)

Reply:
For all the models or `OAuthDeviceCode` and `OAuthDiscoveryDocument`?

Reply:
For all models if possible. As long as these types don't have mutable state there is no reason for them to be `class`es.

Reply:
Done